### DISCUSSION TITLE
Running cQube locally

### DISCUSSION THREAD

**Discussion by GitVivekHub** [Sunbird-cQube/community#780] [Q&A]
Im trying to run cqube locally  and tryiing to hit its national program api with the url http://localhost:3000/ingestion/national_programs.

I have make sure that the Post body that i am sending is correct.

Somehow this api sends back 404 error.

**Comment by RNarmatha** [comment]
At present ,we can install the cQube locally but ingestion is not happen in local. Currently we are working on this task .Will keep you posted on this
